### PR TITLE
:hammer_and_wrench: :mag:Test the CI pipeline

### PR DESCRIPTION
**Pull request by taslim-a-hussain**
new file:   .github/workflows/ci-cd.yml

### CODE REVIEW

File: .github/workflows/ci-cd.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    pull_request:
+        branches: [ main ]
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                node-version: [ 20.10.0 ]
```
Comment (taz-sen) [inline .github/workflows/ci-cd.yml:15]: Add node-versions: [14.x, 16.x, 18.x,  20.x]

File: .github/workflows/ci-cd.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: SoleInk CI/CD
+
+on:
+    push:
+        branches: [ main ]
+    pull_request:
+        branches: [ main ]
+
```
Comment (taz-sen) [inline .github/workflows/ci-cd.yml:8]: Allow running test manually via Action tab.

Comment (taslim-a-hussain) [reply .github/workflows/ci-cd.yml:8]: Resolved

File: .github/workflows/ci-cd.yml (hunk 1 above)
Comment (taslim-a-hussain) [reply .github/workflows/ci-cd.yml]: Resolved

**Review by taz-sen — APPROVED** [review]

**Pull request merged**